src/modules/ekf2/ekf2_params.c

```diff
@@ -1161,6 +1157,26 @@ PARAM_DEFINE_FLOAT(EKF2_RNG_A_HMAX, 5.0f);
  */
 PARAM_DEFINE_FLOAT(EKF2_RNG_A_IGATE, 1.0f);
 
+/**
+ * Gate size for vision velocity estimate fusion
+ * Sets the number of standard deviations used by the innovation consistency test.
```

Comment:
`
* Gate size for vision velocity estimate fusion
*
 * Sets the number of standard deviations used by the innovation consistency test.
 *
  * @group EKF2
`
add some spaces here